plasmapy/simulation/abstractions.py

```diff
@@ -53,4 +57,142 @@ class AbstractTimeDependentSimulation(AbstractSimulation):
     change at any time.
     """
 
-    pass
+    ...
+
+
+class AbstractNormalizations(ABC):
+    """
+    An abstract base class to represent the normalization constants for
+    systems of equations describing plasmas.
+
+    ... warning::
+        This interface is unstable and is subject to change.
```

Comment:
Oops! My suggestion had an extra dot in it. 🤦🏻‍♂️

```suggestion
    .. warning::
       This interface is unstable and is subject to change.
```